.github/workflows/golangci.yml

```diff
@@ -19,8 +20,18 @@ jobs:
         with:
           go-version: 1.19
       - uses: actions/checkout@v3
+      # On pushes and on pull requests from branches that target main, run all linters.
```

Comment:
Can be an inline script instead tbh (i.e a `run` step that does this using some bash), whatever folks prefer.